src/extensions/default/ESLint/domain.js

```diff
@@ -0,0 +1,132 @@
+/*eslint no-process-env:0*/
+/*global require, exports*/
+
+(function () {
+  'use strict';
+
+  var fs = require('fs');
+  var CLIEngine = require('eslint').CLIEngine;
+  var cli = new CLIEngine();
+  var currentProjectRoot = null;
+  var domainName = 'brackets-eslint';
+  var domainManager = null;
+  var noop = function () {};
+  var globalPackagesAvailable = false;
+
+  function _setProjectRoot(projectRoot) {
+    var opts = { useEslintrc: true };
+    var configPath;
+    var rulesDirPath;
+    var ignorePath;
+
+    if (projectRoot) {
```

Comment:
node in brackets is 0.10, no `fs.access` there: https://nodejs.org/docs/latest-v0.10.x/api/fs.html